src/main/Boards/HexBoard.java

```diff
@@ -134,13 +143,15 @@ public Color getFillColor(int x, int y) {
 	}
 
 	public void playAt(int x, int y){
-		if (!isLegalPlay(x, y) && turn == playerTurn) {
-			System.out.println("Turn: " + turn);
-			System.out.println("playerTurn: " + playerTurn);
+		if ((!isLegalPlay(x, y) && turn == playerTurn)) {
+			JOptionPane.showMessageDialog(null, "Illegal Move! Play at another Location", "Invalid move!", JOptionPane.PLAIN_MESSAGE);
+			return;
+		}
+		
```

Comment:
Both of these outcomes are the same. We could condense this into a single if (!isLegalPlay(x,y)) {}